---
layout: publication
title: MARSI: metabolite analogues for rational strain improvement
authors: Cardoso, J. G. R. and Zeidan, A. A. and Jensen, K. and Sonnenschein, N. and Neves, A. R. and  Herrg√•rd, M. J.
journal: Bioinformatics
volume: 34	
number: 13
pages: 2319-2321
year: 2018
doi: 10.1093/bioinformatics/bty108
icon: mdi-file-document
---
Summary: Metabolite analogues (MAs) mimic the structure of native metabolites, can competitively
inhibit their utilization in enzymatic reactions, and are commonly used as selection tools for
isolating desirable mutants of industrial microorganisms. Genome-scale metabolic models representing
all biochemical reactions in an organism can be used to predict effects of MAs on cellular
phenotypes. Here, we present the metabolite analogues for rational strain improvement (MARSI)
framework. MARSI provides a rational approach to strain improvement by searching for metabolites
as targets instead of genes or reactions. The designs found by MARSI can be implemented by
supplying MAs in the culture media, enabling metabolic rewiring without the use of recombinant
DNA technologies that cannot always be used due to regulations. To facilitate experimental implementation,
MARSI provides tools to identify candidate MAs to a target metabolite from a database
of known drugs and analogues.

Availability and implementation: The code is freely available at https://github.com/biosustain/
marsi under the Apache License V2. MARSI is implemented in Python.
